src/Angular-CSharp.csproj.in

```diff
@@ -10,6 +10,7 @@
     <SpaProxyLaunchCommand>npm start</SpaProxyLaunchCommand>
     <NoDefaultLaunchSettingsFile Condition="'$(ExcludeLaunchSettings)' == 'True'">True</NoDefaultLaunchSettingsFile>
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.WebApplication1</RootNamespace>
+    <ImplicitUsings>true</ImplicitUsings>
```

Comment:
We’ve been using `enable` instead of true in other templates.

Reply:
Ok, I'll update that.